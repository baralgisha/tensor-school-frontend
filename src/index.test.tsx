import { sum, sortAndSplit, addsDivToBody } from './index';

describe('название урока', () => {
  test('сумма 1 и 2', () => {
    expect(sum(1, 2)).toBe(3);
  });

  test('test', () => {
    expect(sortAndSplit('1,5,4')).toStrictEqual(['1', '4', '5']);
  });

  test('test1', () => {
    addsDivToBody('as');

    expect(document.body).toMatchSnapshot();
  });
});
