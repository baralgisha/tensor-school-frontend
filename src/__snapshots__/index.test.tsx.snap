// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`название урока test1 1`] = `
<body>
  <div>
    asd
  </div>
</body>
`;
